crates/node/ops/os/mod.rs

```diff
@@ -72,19 +72,6 @@ where
     Ok(euid)
 }
 
-#[op2(fast)]
-pub fn op_process_abort<P>(state: &mut OpState) -> Result<(), AnyError>
-where
-    P: NodePermissions + 'static,
-{
-    {
-        let permissions = state.borrow_mut::<P>();
-        permissions.check_sys("abort", "node:process.abort()")?;
-    }
-
-    std::process::abort();
-}
-
 #[op2]
```

Comment:
It seems to have been reverted.

Reply:
@nyannyacha btw, we shouldn't expose `process.abort()` right? or API can be there but it should be a no-op.

Reply:
@laktek 

Exposing this function to a user worker without permission checks leaves open the possibility that malicious code could abort the process.

Making this function to no-op as you say is one way to go.